content/en/docs/releases/next-release-preview/index.md

```diff
@@ -12,10 +12,12 @@ changelog.
 ## Coming Soon in Release 1.37
 
 ### Orca
-With https://github.com/spinnaker/orca/pull/4804, the api `/applications/{application}/pipelines?expand=false&limit=2` performance is improved when using sql as the backend. 
+With https://github.com/spinnaker/orca/pull/4804, the api `/applications/{application}/pipelines?expand=false&limit=2` performance is improved when using sql as the backend.
 Orca's queries to its sql db are optimized while handling the data obtained from front50.
 TaskController is refactored to support externalized config properties.
 
+https://github.com/spinnaker/orca/pull/4825 adds a new configuration property to orca: `echo.events.ignoreTaskEvents` that defaults to false.  When true, orca doesn't send task events.
```

Comment:
`When true, orca doesn't send task events.` 

vs:

`When true, echo event feeds will not include orca "orchesration" events like saving an application or similar task operations. See  `https://spinnaker.io/docs/setup/other_config/features/notifications/#add-a-listening-webhook-to-spinnaker` 

or maybe similar. Just some more details on what this is?

Reply:
better? https://github.com/spinnaker/spinnaker.io/pull/477/commits/9686e3d9a1760c8b3cb6d3d9fdbdbef180bcf2c6